docs/products/kafka/kafka-connect/howto/iceberg-sink-connector.md

```diff
@@ -75,6 +74,122 @@ Future updates to the Iceberg sink connector include:
 are not supported in this release.
 :::
 
+### Configure AWS IAM permissions {#configure-aws-iam-permissions}
+
+The Iceberg sink connector requires an IAM role with permissions to access Amazon S3 and
+AWS Glue. These permissions allow the connector to write data to an S3 bucket and manage
+metadata in the AWS Glue catalog.
+
+To set up the required permissions:
+
+1. Create an IAM role in
+   [AWS Identity and Access Management (IAM)](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html)
+   with permissions for Amazon S3 and AWS Glue.
+1. Attach the following policy to the IAM role:
+
+   ```json
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Sid": "S3Access",
+         "Effect": "Allow",
+         "Action": [
+           "s3:GetObject",
+           "s3:PutObject",
+           "s3:DeleteObject",
+           "s3:ListBucket",
+           "s3:GetBucketLocation",
+           "s3:AbortMultipartUpload",
+           "s3:ListMultipartUploadParts"
+         ],
+         "Resource": [
+           "arn:aws:s3:::<your-bucket-name>/*"
+         ]
+       },
+       {
+         "Sid": "S3ListBucket",
+         "Effect": "Allow",
+         "Action": "s3:ListBucket",
+         "Resource": [
+           "arn:aws:s3:::<your-bucket-name>"
+         ]
+       },
+       {
+         "Sid": "GlueAccess",
+         "Effect": "Allow",
+         "Action": [
+           "glue:CreateDatabase",
+           "glue:GetDatabase",
+           "glue:GetTables",
+           "glue:SearchTables",
+           "glue:CreateTable",
+           "glue:UpdateTable",
+           "glue:GetTable",
+           "glue:BatchCreatePartition",
+           "glue:CreatePartition",
+           "glue:UpdatePartition",
+           "glue:GetPartition",
+           "glue:GetPartitions"
+         ],
+           "Resource": [
+             "arn:aws:glue:<your-aws-region>:<your-aws-account>:catalog",
+             "arn:aws:glue:<your-aws-region>:<your-aws-account>:database/*",
+             "arn:aws:glue:<your-aws-region>:<your-aws-account>:table/*"
+           ]
+         }
+       ]
+     }
+   ```
+
+   Replace the placeholder values in the policy:
+
+   - `<your-aws-region>`: Your AWS Glue catalog’s region
+   - `<your-aws-account>`: Your AWS account ID
+   - `<your-bucket-name>`: The name of your Amazon S3 bucket
+
+1. Obtain the access key ID and secret access key for this role.
+1. Add these credentials to the Iceberg sink connector configuration.
+
+For more information on creating and managing AWS IAM roles and policies, see the
+[AWS IAM documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html).
+
+### AWS Glue naming conventions
+
+When creating databases and tables in AWS Glue for the Iceberg sink connector,
+follow these naming conventions to ensure compatibility:
+
+- **Database names**:
+  - Must use only lowercase letters (a–z), numbers (0–9), and underscores (_)
+  - Must be between 1 and 255 characters long
```

Comment:
Looks like for Glue the length is 252 max https://github.com/apache/iceberg/blob/main/aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java#L87